2024-04-05 22:36:15,004 - distributed.nanny - INFO -         Start Nanny at: 'tcp://172.16.3.168:36552'
2024-04-05 22:36:16,933 - distributed.worker - INFO -       Start worker at:   tcp://172.16.3.168:38181
2024-04-05 22:36:16,934 - distributed.worker - INFO -          Listening to:   tcp://172.16.3.168:38181
2024-04-05 22:36:16,934 - distributed.worker - INFO -           Worker name:             SLURMCluster-3
2024-04-05 22:36:16,934 - distributed.worker - INFO -          dashboard at:         172.16.3.168:42643
2024-04-05 22:36:16,934 - distributed.worker - INFO - Waiting to connect to:    tcp://172.16.3.67:44553
2024-04-05 22:36:16,934 - distributed.worker - INFO - -------------------------------------------------
2024-04-05 22:36:16,934 - distributed.worker - INFO -               Threads:                         36
2024-04-05 22:36:16,934 - distributed.worker - INFO -                Memory:                  93.13 GiB
2024-04-05 22:36:16,934 - distributed.worker - INFO -       Local Directory: /tmp/dask-worker-space/worker-hxefi5xg
2024-04-05 22:36:16,934 - distributed.worker - INFO - -------------------------------------------------
2024-04-05 22:36:16,953 - distributed.worker - INFO -         Registered to:    tcp://172.16.3.67:44553
2024-04-05 22:36:16,953 - distributed.worker - INFO - -------------------------------------------------
2024-04-05 22:36:16,955 - distributed.core - INFO - Starting established connection to tcp://172.16.3.67:44553
2024-04-05 22:37:13,917 - distributed.core - INFO - Removing comms to tcp://172.16.3.69:34879
2024-04-05 22:38:26,589 - distributed.core - INFO - Removing comms to tcp://172.16.3.133:43428
2024-04-05 22:39:34,192 - distributed.core - INFO - Removing comms to tcp://172.16.3.169:43697
2024-04-05 22:39:43,474 - distributed.core - INFO - Connection to tcp://172.16.3.67:44553 has been closed.
2024-04-05 22:39:43,475 - distributed.worker - INFO - Stopping worker at tcp://172.16.3.168:38181. Reason: worker-handle-scheduler-connection-broken
2024-04-05 22:39:43,476 - distributed.worker - ERROR - Failed to communicate with scheduler during heartbeat.
Traceback (most recent call last):
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/tornado/iostream.py", line 861, in _read_to_buffer
    bytes_read = self.read_from_fd(buf)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/tornado/iostream.py", line 1116, in read_from_fd
    return self.socket.recv_into(buf, len(buf))
ConnectionResetError: [Errno 104] Connection reset by peer

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/worker.py", line 1237, in heartbeat
    response = await retry_operation(
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/utils_comm.py", line 434, in retry_operation
    return await retry(
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/utils_comm.py", line 413, in retry
    return await coro()
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/core.py", line 1269, in send_recv_from_rpc
    return await send_recv(comm=comm, op=key, **kwargs)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/core.py", line 1028, in send_recv
    response = await comm.read(deserializers=deserializers)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/comm/tcp.py", line 241, in read
    convert_stream_closed_error(self, e)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc.__class__.__name__}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.heartbeat_worker local=tcp://172.16.3.168:42566 remote=tcp://172.16.3.67:44553>: ConnectionResetError: [Errno 104] Connection reset by peer
2024-04-05 22:39:43,486 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://172.16.3.168:36552'. Reason: worker-handle-scheduler-connection-broken
2024-04-05 22:39:43,489 - distributed.nanny - INFO - Worker closed
2024-04-05 22:39:45,495 - distributed.nanny - ERROR - Worker process died unexpectedly
2024-04-05 22:39:45,797 - distributed.nanny - INFO - Closing Nanny at 'tcp://172.16.3.168:36552'. Reason: nanny-close-gracefully
2024-04-05 22:39:45,798 - distributed.dask_worker - INFO - End worker
